lib/Raven/Client.php

```diff
@@ -281,26 +281,6 @@ public function removeProcessor(ProcessorInterface $processor)
         $this->processorRegistry->removeProcessor($processor);
     }
 
-    /**
-     * Gets the representation serialier.
-     *
-     * @return ReprSerializer
-     */
-    public function getReprSerializer()
```

Comment:
Removed because I agree with @ste93cry as in https://github.com/getsentry/sentry-php/pull/454#discussion_r115219371